Solution1/ConsoleApp/AutoMap/WeatherProfile.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusinessLayer.DTOs;
+using DataAccessLayer.Model;
+
+namespace ConsoleApp.AutoMap
+{
+    public class WeatherProfile : AutoMapper.Profile
```

Comment:
add using for AutoMapper, and write just Profile

Reply:
+